drivers/mtd_spi_nor/mtd_spi_nor.c

```diff
@@ -362,30 +362,28 @@ static inline void wait_for_write_complete(const mtd_spi_nor_t *dev, uint32_t us
         i++;
 #if IS_USED(MODULE_ZTIMER_USEC)
         if (us) {
-            ztimer_sleep(ZTIMER_USEC, us);
+            uint32_t s = us / div;
```

Comment:
please use descriptive variable names, e.g. `step` instead of `s`

Reply:
went with wait_us